### ISSUE TITLE
FirefoxCSS Installer

### ISSUE THREAD

**Issue by quininer** [FirefoxCSS-Store/FirefoxCSS-Store.github.io#69]
**Is your feature request related to a problem? Please describe.**
The cumbersome installation steps are one of the reasons that prevent people from using `userChrome.css`.
It would be helpful to have an theme installer.

**Describe the solution you'd like**
I made a installer prototype, currently it only works on Linux, but it is easy to make it support other platforms.

If you are interested, I can move the repo to FirefoxCSS-Store and improve it together.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Neikon** [comment]
> 
> 
> **Is your feature request related to a problem? Please describe.**
> The cumbersome installation steps are one of the reasons that prevent people from using `userChrome.css`.
> It would be helpful to have an theme installer.
> 
> **Describe the solution you'd like**
> I made a [installer prototype](https://github.com/quininer/firefox-chrome-manager), currently it only works on Linux, but it is easy to make it support other platforms.
> 
> If you are interested, I can move the repo to FirefoxCSS-Store and improve it together.
> 
> **Describe alternatives you've considered**
> N/A
> 
> **Additional context**
> N/A

Sorry for taking so long to reply. I'm fine with anything that makes the UX as easy as possible. I can't try your installer right now because I only have windows.

I would like to hear your opinion. @andreasgrafen @BeyondMagic

**Comment by andreasgrafen** [comment]
I like the idea of an installer. There have seen similar projects in the past, but most of them are not maintained. I'll give this a shot tho. ^—^

Personally I use a really simple script that just symlinks stuff based on the input I specify. But this only works if the folder structure is setup in a specific way ofc. :D

**Comment by quininer** [comment]
Thank response, I will take time to make it work on macOS and Windows this week.

**Comment by BeyondMagic** [comment]
> use a really simple script that just symlinks stuff based on the input I specify. But this only works if the folder structure is setup in a specific way ofc. :D

Yeah, I think the same, symlinking is definitely the easiest one, you can just kinda edit from anywhere you want and change the linked files to any other theme to test it out.

> I made a installer prototype, currently it only works on Linux, but it is easy to make it support other platforms.

It looks perfect and it's as easy it can be to use, nice.

**Comment by Neikon** [comment]
https://github.com/CosmoCreeper/Sine
